deploy/snapshot/internal/criu/restore_images.go

```
@@ -0,0 +1,140 @@
+
+func prepareRestoreImageDirForSocketScope(checkpointPath string, restoreSocketScope uint64) (string, func(), error) {
+	checkpointPath, err := filepath.Abs(checkpointPath)
+	if err != nil {
+		return "", nil, fmt.Errorf("failed to resolve checkpoint path: %w", err)
+	}
+
+	filesImage, err := os.Open(filepath.Join(checkpointPath, filesImageFilename))
+	if err != nil {
+		return "", nil, fmt.Errorf("failed to open %s: %w", filesImageFilename, err)
+	}
+	defer filesImage.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `filesImage.Close` is not checked

(errcheck)